src/line_absorption.jl

```diff
@@ -258,17 +271,15 @@ function sigma_line(λ::Real)
 end
 
 """
-    line_profile(λ₀, invΔλ_D, Δλ_L, line_amplitude, λ)
+    line_profile(λ₀, σ, γ, amplitude, λ)
 
-A voigt profile centered on λ₀ with Doppler width 1/`invΔλ_D` and Lorentz width `Δλ_L` 
-evaluated at `λ` (cm).  Note that this returns values in units of cm^-1.
+A voigt profile centered on λ₀ with Doppler width σ (NOT √[2] σ, as the "Doppler width" is often 
+defined) and Lorentz HWHM γ evaluated at `λ` (cm).  Returns values in units of cm^-1.
 """
-function line_profile(λ₀::Real, invΔλ_D::Real, Δλ_L::Real, line_amplitude::Real, λ::Real)
-    _line_profile(λ₀, invΔλ_D, Δλ_L*invΔλ_D/(4π), line_amplitude*invΔλ_D/sqrt(π), λ)
-end
-function _line_profile(λ₀::Real, invΔλ_D::Real, Δλ_L_invΔλ_D_div_4π::Real, 
-                       amplitude_invΔλ_D_div_sqrt_π::Real, λ::Real)
-    voigt(Δλ_L_invΔλ_D_div_4π, abs(λ-λ₀) * invΔλ_D) * amplitude_invΔλ_D_div_sqrt_π
+function line_profile(λ₀::Real, σ::Real, γ::Real, amplitude::Real, λ::Real)
+    inv_σsqrt2 = 1/(σ*sqrt(2))
+    scaling = inv_σsqrt2 / sqrt(π) * amplitude
+    voigt_hjerting(γ*inv_σsqrt2, abs(λ-λ₀)*inv_σsqrt2) * scaling
 end
```

Comment:
```suggestion
function line_profile(λ₀::Real, σ::Real, γ::Real, amplitude::Real, λ::Real)
    inv_σsqrt2 = 1/(σ*sqrt(2))
    inv_sqrtπ = 1.0 / sqrt(π)
    scaling = inv_σsqrt2 * inv_sqrtπ * amplitude
    voigt_hjerting(γ*inv_σsqrt2, abs(λ-λ₀)*inv_σsqrt2) * scaling
end
```

Here is a minor optimization that shouldn't hurt readability and should convert divsion operations to multiplications.

As I recall, `Julia` is "smart" enough to precompute `inv_sqrtπ` when compiling `line_profile`. With that said, you might need to instead do something like the following (which would be necessary in just about any other language):
```julia
const inv_sqrtπ = 1.0 / sqrt(π)
function line_profile(λ₀::Real, σ::Real, γ::Real, amplitude::Real, λ::Real)
    inv_σsqrt2 = 1/(σ*sqrt(2))
    scaling = inv_σsqrt2 * inv_sqrtπ * amplitude
    voigt_hjerting(γ*inv_σsqrt2, abs(λ-λ₀)*inv_σsqrt2) * scaling
end
```

As an aside, the calculations for H-lines could probably be slightly optimized if `line_profile` function accepted `inv_σ` or `inv_σsqrt2` instead of `σ`. I'm sure you're well-aware of this, and I suspect this may not not be worth doing.

Reply:
>As an aside, the calculations for H-lines could probably be slightly optimized if line_profile function accepted inv_σ or inv_σsqrt2 instead of σ. I'm sure you're well-aware of this, and I suspect this may not not be worth doing.

I remember that is is true from when se optimized the previous code, but I don't remember why.